script.js

```diff
@@ -1,20 +1,236 @@
+           <li class="lang">${project.technologies.tech2}</li>
+           <li class="lang JavaScript">${project.technologies.tech3}</li>
+       </ul>
+
+       <div class="primarytext4">
+           <button class="project" id =${project.id}>${project.details}</button>
+       </div>
+   </div>
+</div>`;
+  cardSection.append(cardDiv);
+});
+
+const seeproject = document.querySelectorAll('.project');
+const overlay = document.getElementById('overCover');
+const popup = document.querySelector('.popup');
+const over = document.querySelector('.over');
+seeproject.forEach((p) => p.addEventListener('click', (p) => {
+  const { id } = p.target;
+  const pop = projects.find((p) => p.id === id);
+  popup.innerHTML = `
+  <div class="titles">
+   <div class="tile">
+  <h2 class="primarytext-title must-ignor1">${pop.title}</h2>
+  <h2 class="primarytext-title must-ignor2">${pop.title1}</h2>
+   <button class="close">&times;</button>
+    </div>
+    <ul class="social-icons10">
+    <li class="cano must-ignor1">${pop.company}</li>
+    <li class="cano must-ignor2">${pop.company1}</li>
+    <li class="must-ignor1" ><img src="${pop.icon}" alt="dot">&nbsp; ${pop.specialization}</li>
+    <li class="must-ignor2" ><img src="${pop.icon}" alt="dot">&nbsp; ${pop.specialization}</li>
+    <li class="must-ignor1" ><img src="${pop.icon}" alt="dot">&nbsp; ${pop.year}</li>
+    <li class="must-ignor2" ><img src="${pop.icon}" alt="dot">&nbsp; ${pop.year}</li>
+  </ul>
+  
+  <img src="${pop.imageMobile}" alt="Tonic-image" class="imageMobile must-ignor1">
+  <img src="${pop.imageDesktop}" alt="Nature-image" class="imageDesktop must-ignor2">
+  <div class="explain">
+    <p class="tonic-text">
+      ${pop.description1} 
+      <br>
+      <br>
+      <span class="must-ignor2"> ${pop.description1}</span>
+    </p>
+    <ul class="primarytext3 techno">
+    <li class="lang">${pop.technologies.tech1}</li>
+    <li class="lang">${pop.technologies.tech2}</li>
+    <li class="lang JavaScript">${pop.technologies.tech3}</li>  
+    </ul>
+    
+        <hr class="invisable">
+        <div class="btns">
+          <button type="button" id="live" class="but">See live &nbsp; &nbsp;
+            <img src="./Image/live.png" alt="live">
+          </button>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
+          <button type="button" id="source" class="but">See source &nbsp; &nbsp;
+            <img src="./Image/github.svg" alt="source">
+          </button>
+        </div>
```

Comment:
- [ ] Kindly consider fixing the design for buttons to match with design guidelines in Figma 

| Your design  | Design guidline |
| ------------- | ------------- |
| ![Screenshot 2023-04-12 122744](https://user-images.githubusercontent.com/53402600/231422828-b4356b76-3b70-49ae-831b-7832e22990d7.png) | ![image](https://user-images.githubusercontent.com/53402600/231422932-aabfd306-7712-44ca-9b6b-7446ca8970f2.png) |